teaspoons-cli/teaspoons/cli.py

```diff
@@ -1,14 +1,29 @@
 # cli.py
 
-import typer
-from commands.auth import auth_app
-from commands.pipelines import pipelines_app
+import click
+import logging
 
-cli = typer.Typer()
-cli.add_typer(auth_app, name="auth", help="Authentication commands")
-cli.add_typer(pipelines_app, name="pipelines", help="Pipeline commands")
+from teaspoons import __version__
+from teaspoons.commands.auth_commands import auth
+from teaspoons.commands.pipelines_commands import pipelines
+
+
+# Context settings for commands, for overwriting some click defaults
+CONTEXT_SETTINGS = dict(help_option_names=["-h", "--help"])
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
is this necessary if we arent logging in the file (or are we logging and i dont see it)?